lib/inputstreamhelper/widevine/arm.py

```diff
@@ -137,10 +137,13 @@ def install_widevine_arm(backup_path):
     # Google will remove support for older Widevine CDM's on May 31, 2021
     # More info at https://github.com/xbmc/inputstream.adaptive/issues/678 and https://www.widevine.com/news
 
-    # Experimental: Check for TCMalloc support
-    tcmalloc_path = '/usr/lib/libtcmalloc_minimal.so'
+    # Experimental: Check if TCMalloc library exists and gets preloaded
+    libtcmalloc = 'libtcmalloc_minimal.so'
```

Comment:
Since CoreELEC allows to link to libtcmalloc too, I would not make this too strict. Chances that some other (non-tcmalloc) library includes `tcmalloc` is pretty improbable.
```suggestion
    libtcmalloc = 'libtcmalloc'
```